### PR TITLE
Removed unused styles from old CSS

### PR DESCRIPTION
**Pull request by dereksmart**
**_Originally opened and submitted by @MichaelArestad in jetpack-react_**

This was pretty painful. The rest of the styles are in use somewhere and we'll remove them as we retire their correlating sections.

Not many visual changes. The removal of global border-box tweaked some padding, but that's about it, hopefully.

### CODE REVIEW

File: scss/templates/_connection-landing.scss [hunk 1]
```diff
@@ -11,22 +11,6 @@ $j-gutter: em(20px);
 $landing-bdr: #D9D9D9;
```
Comment (jeffgolenski) [inline scss/templates/_connection-landing.scss]: With the new connection landing page, I'm sure we'll be able to remove most, if not all, "old" connection scss (this entire file)

File: scss/_utilities/_grid.scss [hunk 2]
```diff
@@ -6,10 +6,10 @@
 .j-row {
```
Comment (jeffgolenski) [inline scss/_utilities/_grid.scss]: As far as I remember, the only thing within jetpack using this grid is the settings areas, so we'll be able to eliminate this. grid system.

File: scss/templates/_connection-landing.scss (hunk 1 above)
Comment (MichaelArestad) [inline scss/templates/_connection-landing.scss]: I agree. I am, however, only removing CSS that I 100% for sure know is not used in either the old or new versions. I will make another PR as we start removing old sections.

File: scss/templates/_connection-landing.scss [hunk 3]
```diff
@@ -359,14 +359,6 @@ So I moved to stack the svgs as actual imgs instead. IE also had a hard time dea
```
Comment (jeffgolenski) [inline scss/templates/_connection-landing.scss]: Will be able to remove this entire thing as well.

File: scss/templates/_connection-landing.scss (hunk 1 above)
Comment (jeffgolenski) [inline scss/templates/_connection-landing.scss]: Perfect!

**Comment by jeffgolenski** [comment]
@MichaelArestad  IPF. :shipit:

**Pull request merged**